DOCS_ENGINEERING.md

```
@@ -0,0 +1,63 @@
+# Zen Eyer Project: Engineering Standards & AI Directives
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)